lightblock.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/wire"
+)
+
+type LightBlock struct {
+	height  int32
+	header  *wire.BlockHeader
+	lcStore BTCLightClientStorage
+}
```

Comment:
Could you double check if there is already a struct defined for SPV blocks? I'm pretty sure this should be defined somewhere in `btcd`. What babylon is using?

Reply:
Yes. Actually, Babylon use bitcoin-spv logic(This only for check merkle tree) and make it compatible with btcd.